packages/contracts/src/2.0.0/utils/SafeMath/SafeMath.sol

```diff
@@ -7,8 +7,14 @@ contract SafeMath {
         pure
         returns (uint256)
     {
+        if (a == 0) {
+            return 0;
+        }
         uint256 c = a * b;
-        assert(a == 0 || c / a == b);
+        require(
+            c / a == b,
+            "OVERFLOW"
```

Comment:
How about `"INTEGER_OVERFLOW"` or `"UINT256_OVERFLOW"` so as to not be confused with things like buffer overflow or stack overflow?